channels/rdpgfx/client/rdpgfx_main.c

```diff
@@ -1750,6 +1806,10 @@ UINT DVCPluginEntry(IDRDYNVC_ENTRY_POINTS* pEntryPoints)
 		context->GetSurfaceData = rdpgfx_get_surface_data;
 		context->SetCacheSlotData = rdpgfx_set_cache_slot_data;
 		context->GetCacheSlotData = rdpgfx_get_cache_slot_data;
+
+		// TODO: move somewhere else around this func?
```

Comment:
I think that's a find location, because all callbacks are being initialized there. Where did you think to put it?

Reply:
lets keep here